model/log_iterator.go

```diff
@@ -349,9 +349,10 @@ type mergingIterator struct {
 
 // NewMergeIterator returns a LogIterator that merges N buildlogger logs,
 // passed in as LogIterators, respecting the order of each line's timestamp.
-func NewMergingIterator(ctx context.Context, iterators ...LogIterator) LogIterator {
+func NewMergingIterator(ctx context.Context, reverse bool, iterators ...LogIterator) LogIterator {
```

Comment:
could we just add a .Reverse() method to the log iterator? 

bool arguments to functions always feel a bit messy and tend to proliferate.

Reply:
what if instead I create a separate constructor functions, for example, `NewReverseMergingIterator`?